.gitignore

```diff
@@ -8,6 +8,7 @@ collections/ansible_collections/*
 !collections/ansible_collections/cloudkit
 collections/ansible_collections/cloudkit/*
 !collections/ansible_collections/cloudkit/global_config
```

Comment:
I think those things are left overs from the time I did create the repo, I think we can delete